### DISCUSSION TITLE
Errors "file access denied (cmd.StorageErr)"

### DISCUSSION THREAD

**Discussion by MiroslavVanicky** [minio/minio#17825] [Q&A]
Hi, 

I just downloaded latest Minio today (RELEASE.2023-08-04T17-40-21Z (go1.19.12 windows/amd64)).

These errors are written to console:
![image](https://github.com/minio/minio/assets/46671359/e36048de-43c7-45bc-ac12-0ce29aafb75f)

Does anyone have any solution for this?
Thanks,

Mirek

**Comment by harshavardhana** [comment]
Don't use windows @MiroslavVanicky use Linux.

**Reply by MiroslavVanicky** [reply]
Why? Windows version is not working fully? This is some known issue?
I am developing on Windows machine and need to test with Minio locally, so Windows version seemed best for me.

**Reply by klauspost** [reply]
@harshavardhana This could indicate a missing lock or another type of race. This generally shouldn't happen, unless something outside of MinIO is touching the files.

@MiroslavVanicky Could you please turn off *all* antivirus scanning and AV and see if it still reproduces.

Some AV software keep locks on files after they have been written while they are scanning it. That could be the cause of this error.

**Reply by MiroslavVanicky** [reply]
I tried, we have no other AV software than this built-in Windows Security and we put folder exclusion on Minio data folder & folder where minio.exe is. No success. 
Could you recommend any way how can I find process that is touching minio files?
